bin/support/sh/functions/cic.sh

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
+if [ "${TRACKS_PATH}" == "" ]
+then
+    TRACKS_PATH="${CURRENT_PATH}/../tracks"
+fi
+
```

Comment:
We can use bash's default varaible features here and avoid an if statement by using:  `TRACKS_PATH=${TRACKS_PATH:-${CURRENT_PATH}/../tracks}`  

instead... and likewise for EXERCISES_PATH below too.

Reply:
Thanks, this is really useful!